src/System.Collections/tests/Performance/Perf.Dictionary.cs

```diff
@@ -20,8 +20,7 @@ public class Perf_Dictionary
             while (dict.Count < size)
             {
                 int key = rand.Next(500000, int.MaxValue);
-                if (!dict.ContainsKey(key))
-                    dict.Add(key, 0);
+                dict.TryAdd(key, 0);
```

Comment:
I agree, and I think that should be part of a different PR, as this one is meant to set a common baseline to release/2.0.0 and master.